pkg/detectors/aws/aws.go

```diff
@@ -101,6 +103,34 @@ func GetHMAC(key []byte, data []byte) []byte {
 	return hasher.Sum(nil)
 }
 
+func GetAccountNumFromAWSID(AWSID string) (string, error) {
```

Comment:
suggestion: Lets go ahead and also make this function private since it probably won't be used outside this pkg?

Reply:
I'm going to move it into a common lib, since it will be used in the /aws and /awssessionkey packages.